--- viper.py~	2009-09-28 10:48:56.000000000 +0200
+++ viper.py	2009-10-06 13:39:05.000000000 +0200
@@ -797,7 +797,7 @@
             renWin = vtk.vtkRenderWindow()
             renWin.SetWindowName(self.title)
             renWin.AddRenderer(ren)
-            renWin.SetSize(self.window_size)
+            renWin.SetSize(600, 400)
         return ren, renWin
 
     def compute_vector_scale(self, x, scale=2.0):
