diff -Nru slepc-3.1-p4.orig/config/configure.py slepc-3.1-p4/config/configure.py
--- slepc-3.1-p4.orig/config/configure.py	2010-09-25 09:07:36.000000000 +0200
+++ slepc-3.1-p4/config/configure.py	2010-11-07 08:42:59.000000000 +0100
@@ -214,8 +214,8 @@
   slepcrules = open(os.sep.join([confdir,'slepcrules']),'w')
 except:
   sys.exit('ERROR: cannot create rules file in ' + confdir)
-if prefixinstall and os.path.isfile(os.sep.join([prefixdir,'include','slepc.h'])):
-  sys.exit('ERROR: prefix directory ' + prefixdir + ' contains files from a previous installation')
+#if prefixinstall and os.path.isfile(os.sep.join([prefixdir,'include','slepc.h'])):
+#  sys.exit('ERROR: prefix directory ' + prefixdir + ' contains files from a previous installation')
 
 # Open log file
 log.Open(os.sep.join([confdir,'configure.log']))
diff -Nru slepc-3.1-p4.orig/configure slepc-3.1-p4/configure
