diff -Nru trilinos-10.12.2-Source.orig/cmake/TPLs/FindTPLUMFPACK.cmake trilinos-10.12.2-Source/cmake/TPLs/FindTPLUMFPACK.cmake
--- trilinos-10.12.2-Source.orig/cmake/TPLs/FindTPLUMFPACK.cmake	2012-10-19 10:42:09.862574168 +0200
+++ trilinos-10.12.2-Source/cmake/TPLs/FindTPLUMFPACK.cmake	2012-10-19 10:40:21.546577999 +0200
@@ -55,7 +55,14 @@
 
 INCLUDE(TribitsTplDeclareLibraries)
 
-TRIBITS_TPL_DECLARE_LIBRARIES( UMFPACK
-  REQUIRED_HEADERS umfpack.h amd.h UFconfig.h
-  REQUIRED_LIBS_NAMES umfpack amd
-  )
+if (APPLE)
+  TRIBITS_TPL_DECLARE_LIBRARIES( UMFPACK
+    REQUIRED_HEADERS umfpack.h amd.h UFconfig.h
+    REQUIRED_LIBS_NAMES umfpack amd SuiteSparse
+    )
+else()
+  TRIBITS_TPL_DECLARE_LIBRARIES( UMFPACK
+    REQUIRED_HEADERS umfpack.h amd.h UFconfig.h
+    REQUIRED_LIBS_NAMES umfpack amd
+    )
+endif()
diff -Nru trilinos-10.12.2-Source.orig/packages/PyTrilinos/src/CMakeLists.txt trilinos-10.12.2-Source/packages/PyTrilinos/src/CMakeLists.txt
--- trilinos-10.12.2-Source.orig/packages/PyTrilinos/src/CMakeLists.txt	2012-10-19 10:42:10.098574160 +0200
+++ trilinos-10.12.2-Source/packages/PyTrilinos/src/CMakeLists.txt	2012-10-19 10:40:21.546577999 +0200
@@ -195,6 +195,10 @@
   PROPERTIES LINK_FLAGS "${EXTRA_LINK_ARGS}"
   )
 
+if (APPLE)
+  TARGET_LINK_LIBRARIES(pytrilinos ${PYTHON_LIBRARIES})
+endif()
+
 #
 # Define the PyTrilinos swig setup
 SET(CMAKE_SWIG_FLAGS "")
diff -Nru trilinos-10.12.2-Source.orig/packages/zoltan/src/zz/murmur3.c trilinos-10.12.2-Source/packages/zoltan/src/zz/murmur3.c
--- trilinos-10.12.2-Source.orig/packages/zoltan/src/zz/murmur3.c	2012-10-19 10:42:10.070574161 +0200
+++ trilinos-10.12.2-Source/packages/zoltan/src/zz/murmur3.c	2012-10-19 10:41:09.706576296 +0200
@@ -12,7 +12,13 @@
 /*----------------------------------------------------------------------------*/
 /* Platform-specific functions and macros */
 
+#include <stdint.h>
+
+#if __GNUC__ && __GNUC_STDC_INLINE__
+ #define FORCE_INLINE inline __attribute__((always_inline))
+#else
 #define FORCE_INLINE __attribute__((always_inline))
+#endif
 
 /* KDDKDD 
 inline uint32_t rotl32 ( uint32_t x, int8_t r )
@@ -42,7 +48,7 @@
 /*--------------------------------------------------------------------------- */
 /* Finalization mix - force all bits of a hash block to avalanche */
 
-FORCE_INLINE uint32_t fmix32 ( uint32_t h )
+static FORCE_INLINE uint32_t fmix32 ( uint32_t h )
 {
   h ^= h >> 16;
   h *= 0x85ebca6b;
@@ -55,7 +61,7 @@
 
 /*---------- */
 
-FORCE_INLINE uint64_t fmix64 ( uint64_t k )
+static FORCE_INLINE uint64_t fmix64 ( uint64_t k )
 {
   k ^= k >> 33;
   k *= BIG_CONSTANT(0xff51afd7ed558ccd);
